MJxPrep.js

```diff
@@ -12,12 +12,17 @@
 // Make sure that this script only loads once
 if (window.MJxPrep) {
 } else {
+  // Specify options
+  window.MJxPrepOptions = {
+    'conj_as_star': true
```

Comment:
Just checking: We are expecting people to ...

1. decide on course-wide options for these things
2. download the js file
3. customize the js file
4. include at appropriate places in their course

Is that correct? I was thinking about this, and that's the best I could come up with.

Reply:
Pretty much. Note that including at appropriate locations typically means loading it as a preprocessor for a textbox. Without edX making anything useful, I think this is the best we've got.